hw3/02-url.js

```diff
@@ -23,13 +32,29 @@ const server = http.createServer((req, res) => {
     return result;
   };
 
-  if (req.url === '/') {
+  if (req.url === "/") {
     let routeResults = getRoutes();
 
-    res.writeHead(200, { 'Content-Type': 'text/html' });
+    res.writeHead(200, { "Content-Type": "text/html" });
     res.write(`<h1>Exercise 02</h1>`);
 
     res.write(`<ul> ${routeResults} </ul>`);
+  } else {
+    res.writeHead(200, { "Content-Type": "text/html" });
+    const queryObj = url.search;
+    const params = new URLSearchParams(queryObj);
+    const map = new Map();
+    params.forEach((value, key) => {
+      map.set(key, value);
+    });
+    let table = "<table border=2>";
+    let data = Array.from(map);
+    for (const [key, value] of map) {
+      table += `<tr><td>${key}</td><td>${value}</td><tr>`;
+    }
+    table += "</table>";
+    res.write(table);
+    res.end();
   }
```

Comment:
- Some strings in this file could be used with single quotes as the guide recommends,
- Naming Conventions: Good, you cover camelCase and the names are descriptive which makes reading your string interpolations easy.